projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.html

```diff
@@ -40,12 +46,8 @@
         <div class="sap-icon--log">
           <a
             title="{{ 'asm.logout' | cxTranslate }}"
-            *ngIf="
-              (csAgentToken$ | async)?.access_token &&
-              !(customer$ | async) &&
```

Comment:
if we don't need this anymore, should we still have it saved in a variable?

Reply:
Yes we still need them.  Both `customer$` and `csAgentToken$` are used elsewhere in the template.